packages/parser/package.json

```diff
@@ -37,6 +37,38 @@
         "types": "./lib/esm/index.d.ts",
         "default": "./lib/esm/index.js"
       }
+    },
+    "./middleware": {
+      "require": "./lib/cjs/middleware/parser.js",
+      "import": "./lib/esm/middleware/parser.js"
+    },
+    "./schemas": {
+      "require": "./lib/cjs/schemas/index.js",
+      "import": "./lib/esm/schemas/index.js"
+    },
+    "./envelopes": {
+      "require": "./lib/cjs/index.js",
+      "import": "./lib/esm/index.js"
+    }
```

Comment:
We're also missing a `exports.types` to match the `typeVersions.*.types` one.

Reply:
Added types block